mars/api/models.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+from enum import Enum
+
+class Player(models.Model):
+    name = models.charField(max_length = 30)
+
+class 
```

Comment:
Is that an anonymous class of some sort? :D

Reply:
Work in progress :)